source/_components/sensor.rest.markdown

```diff
@@ -153,3 +154,40 @@ sensor:
       User-Agent: Home Assistant REST sensor
 ```
 
+### {% linkable_title Fetch multiple JSON values and present them as attibutes %}
+
+This sample fetches a weather report from OpenWeatherMap.org, maps the
+resulting data into attributes of the RESTful sensor and then creates
+a set of template sensors that monitor the attributes and present the
+values in a usable form.
+
+```
+- platform: rest
+  name: OWM_report
+  json_attributes: main,weather
+  value_template: '{{value_json["weather"][0]["description"].title()}}'
```

Comment:
Like all other templates will this be rendered empty, see https://home-assistant.io/developers/documentation/standards/#templates.

Reply:
Oops. Sorry about that. I shall fix it.